assets/js/modules/thank-with-google/components/setup/SetupCreatePublication.js

```diff
@@ -35,6 +38,15 @@ export default function SetupCreatePublication() {
 		select( MODULES_THANK_WITH_GOOGLE ).getServiceCreatePublicationURL()
 	);
 
+	const viewContext = useViewContext();
+
+	const handleCreatePublication = useCallback( async () => {
+		await trackEvent(
+			`${ viewContext }_thank-with-google`,
+			'create_publication'
+		);
+	}, [ viewContext ] );
```

Comment:
Let's update these `onClick` callbacks to use `navigateTo` to handle the navigation similar to other uses. E.g. (only `await` before is correct) https://github.com/google/site-kit-wp/blob/b3992bf149b3d4fc6c53d28c1dd4d2aeab9b67b5/assets/js/components/EntityHeader.js#L97-L100

Reply:
Hi, we can't use `navigateTo` because in all the cases, the link is supposed to be opened in new tab. For the same reason, I think the event tracking is safe. What do you think?